src/management/database.ts

```diff
@@ -29,6 +29,7 @@ export enum DatabaseInstanceState {
 export enum DatabaseLocation {
   US_CENTRAL1 = "us-central1",
   EUROPE_WEST1 = "europe-west1",
+  ASIAN_SOUTHEAST1 = "asia-southeast1",
```

Comment:
The enum should match the name

```suggestion
  ASIA_SOUTHEAST1 = "asia-southeast1",
```

Reply:
Oh, sorry, meant to keep them the same.